test/integration/PaymentTypes/PaylaterInstallmentTest.php

```diff
@@ -0,0 +1,249 @@
+<?php
+
+/** @noinspection PhpUnhandledExceptionInspection */
+/** @noinspection PhpDocMissingThrowsInspection */
+/**
+ * This class defines integration tests to verify interface and
+ * functionality of the payment method Installment Secured.
+ *
+ * Copyright (C) 2020 - today Unzer E-Com GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @link  https://docs.unzer.com/
+ *
+ * @package  UnzerSDK\test\integration\PaymentTypes
+ */
+
+namespace UnzerSDK\test\integration\PaymentTypes;
+
+use UnzerSDK\Constants\CustomerTypes;
+use UnzerSDK\Exceptions\UnzerApiException;
+use UnzerSDK\Resources\Customer;
+use UnzerSDK\Resources\CustomerFactory;
+use UnzerSDK\Resources\EmbeddedResources\Address;
+use UnzerSDK\Resources\EmbeddedResources\Paylater\InstallmentPlansQuery;
+use UnzerSDK\Resources\EmbeddedResources\Paylater\InstallmentPlan;
+use UnzerSDK\Resources\PaymentTypes\PaylaterInstallment;
+use UnzerSDK\Resources\TransactionTypes\Authorization;
+use UnzerSDK\Resources\TransactionTypes\Cancellation;
+use UnzerSDK\Resources\TransactionTypes\Charge;
+use UnzerSDK\test\BaseIntegrationTest;
+use function count;
+
+class PaylaterInstallmentTest extends BaseIntegrationTest
+{
+    /**
+     * Verify that paylater installment plans can be fetched.
+     * @test
+     */
+    public function installmentPlanShouldBeFetchable(): void
```

Comment:
Maybe you can add some negative tests
I thinks (for example) null or negative amounts are not allowed

Reply:
Its not allowed to set them to null, by parameter defenition, so thats not testable.
But I have added a test with negative amount to verify error behaviour.